README.md

```diff
@@ -323,3 +339,23 @@ exploreUrl =
 * **feedFetcherTimeout**: Maximum number of seconds to wait for an RSS or Atom feed to load. If a feed takes longer than that number of seconds to update, the update will be aborted
 * **useCronUpdates**: To use a custom update/cron script you need to disable the cronjob which is run by Nextcloud by default by setting this to false
 * **exploreUrl**: If given that url will be contacted for fetching content for the explore feed
+
+## Keyboard Shortcuts
```

Comment:
This is available in the app itself. This would require us to duplicate efforts for keeping it up to date so I'd prefer to not include it in the README. Looks good otherwise

Reply:
Yes, I didn't know I can scroll in the settings. Btw I might remember you from YALM magazine?

Reply:
Yep. I worked on the wiki/publishing software there.

Reply:
Hehe, yes me too :joy_cat: